charts/timescaledb-single/Chart.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.10.0
+version: 0.10.1
```

Comment:
Why are you trying to update the chart version? This is up to the maintainers.

Reply:
This is not. We require chart update on each PR as mentioned in contributing doc - https://github.com/timescale/helm-charts/blob/main/CONTRIBUTING.md#technical-requirements